contents/blog/workflows-alpha.md

```diff
@@ -0,0 +1,106 @@
+---
+date: 2025-11-03
+title: Workflows Are Now in Alpha. And I Already Managed to Break Mine
+author:
+  - sara-miteva
+featuredImage: >-
+  
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+---
+
+Monday morning. This is me, happy and caffeinated, walking to my coworking space, feeling productive and ready for the week.
+
+[Add image here, art request is submitted]
+
+Last week I had set up a new automation using our new **workflows** product (soon coming to beta).  
+The goal: test if everything works – and, at the same time, let users know it’s available to try.  
+I sent out a simple test email.  
+It looked like this:
```

Comment:
```suggestion
Last week, I set up a new automation using our new **workflows** product, which is in alpha. I wanted to test how it worked – and, at the same time, let users know it’s available to try.  

I sent out a simple test email that looked like this:
```

I prefer this sentence construction personally. You could go the other way, but you'd need to pad it out with line breaks. Right now it looks like:

<img width="643" height="162" alt="Image" src="https://github.com/user-attachments/assets/a02ad6c8-bff1-4a80-ae98-02cf01ed29fd" />